justfile

```diff
@@ -50,40 +60,122 @@ bootstrap *ARGS:
 @update:
     echo "TODO: update"
 
-# script to rule them all - end
+# --------------------------------------------------
+# script to rule them all recipes - end
+# --------------------------------------------------
 
 # Update the version; Used before release to production
 @bump *ARGS:
     bumpver update {{ ARGS }}
 
-# ???
+# --------------------------------------------------
+# Caddy recipes
+# --------------------------------------------------
+
+# Format our Caddyfile
 @caddy-fmt:
     docker-compose run --rm caddy caddy fmt -overwrite /etc/caddy/Caddyfile
 
-# ???
+# Ir our Caddyfile valid?
```

Comment:
@jefftriplett looks like there's a typo here

`Ir our Caddyfile valid?`

Should be

`Is our Caddyfile valid?`

Reply:
This should be fixed in #907

Reply:
🤦  good catch. I have a huge stash that I trying not to lose while doing this.